### PR TITLE
camera_info_manager_py: 0.2.3-1 in 'melodic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by jordy-van-appeven**
Increasing version of package(s) in repository `camera_info_manager_py` to `0.2.3-1`:

- upstream repository: https://github.com/ros-perception/camera_info_manager_py.git
- release repository: https://github.com/appie-17/camera_info_manager_py-release.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.8.0`
- previous version for package: `null`

## camera_info_manager_py

- No changes

### CODE REVIEW

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -527,6 +527,13 @@ repositories:
       type: git
       url: https://github.com/ros-perception/calibration.git
       version: hydro
+  camera_info_manager_py:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/appie-17/camera_info_manager_py-release.git
```
Comment (clalancette) [inline melodic/distribution.yaml]: In previous releases (like [Kinetic](https://github.com/ros/rosdistro/blob/master/kinetic/distribution.yaml#L1095)), this was released from the ros-perception organization (using the ros-gbp for the actual release).  Why the change?

**Review by clalancette — CHANGES_REQUESTED** [review]
We need to figure out why the release organization changed.

File: melodic/distribution.yaml (hunk 1 above)
Comment (appie-17) [reply melodic/distribution.yaml]: I changed the organization since it does not seem like it is actively maintained anymore for new releases... Since there are no issues running this package on newer releases, it made sense to release the package myself.

Comment (clalancette) [reply melodic/distribution.yaml]: We generally don't hand over maintainership for a package unless explicitly given permission by the original maintainer, or if all attempts to contact the original maintainer go unanswered for a period of time.  Can you document your attempts to contact the original maintainers of the package?

**Comment by tfoote** [comment]
As this is effectively taking over maintenance unannounced I'm going to close it.

Please see some of the discussion in [this thread](https://discourse.ros.org/t/releasing-repositories-form-other-people/1797?u=tfoote) for more info on a better approach pros cons etc.
